src/assets/javascript/analytics/ga4/init.js

```diff
@@ -0,0 +1,45 @@
+/* global window */
+
+window.DI = window.DI || {}
+window.DI.analyticsGa4 = window.DI.analyticsGa4 || {};
+
+(function (analytics) {
+
+  'use strict'
+
+  // container is an optional parameter used for unit tests
```

Comment:
What container parameter are you referring to? `window.DI.analytics.vars.ga4ContainerId`?